docs/testing.md

```diff
@@ -0,0 +1,78 @@
+CRI-Containerd Testing Guide
+============================
+This document assumes you have already setup the development environment (go, git, `cri-contaienrd` repo etc.).  
+ 
+Before sending pull requests you should at least make sure your changes have passed code verification, unit test and CRI validation test.
+## Code verification
```

Comment:
That's why we call em opinions, everyone has one :-) To me it's the verification test for all files code, config, documentation etc.   lint, format, headers, etc for code, yaml, md files, yada yada